mypy/moduleinfo.py

```diff
@@ -21,198 +21,205 @@
 
 third_party_modules = {
     # From Python 3 Wall of Superpowers (https://python3wos.appspot.com/)
-    'simplejson',
-    'requests',
-    'six',
     'pip',
-    'virtualenv',
-    'boto',
-    'setuptools',
-    'dateutil',
-    'pytz',
+    'urllib3',
+    'six',
+    'botocore',
+    'python-dateutil',
+    's3transfer',
+    'pyyaml',
+    'requests',
     'pyasn1',
-    'nose',
-    'yaml',
-    'lxml',
-    'jinja2',
     'docutils',
-    'markupsafe',
-    'paramiko',
-    'Crypto',
+    'jmespath',
+    'certifi',
     'rsa',
-    'django',
-    'coverage',
-    'pika',
+    'setuptools',
+    'idna',
+    'awscli',
+    'futures',
     'colorama',
-    'ecdsa',
-    'psycopg2',
-    'httplib2',
-    'mock',
+    'chardet',
+    'wheel',
+    'simplejson',
+    'boto3',
+    'pytz',
+    'numpy',
+    'markupsafe',
+    'jinja2',
+    'cffi',
+    'cryptography',
+    'protobuf',
+    'awscli-cwlogs',
```

Comment:
This is another PyPI distribution name that isn't a real Python package name. There are some other examples below.